include/aspect/simulator_access.h

```diff
@@ -32,20 +32,6 @@
 #include <aspect/global.h>
 #include <aspect/parameters.h>
 #include <aspect/introspection.h>
-#include <aspect/material_model/interface.h>
-#include <aspect/geometry_model/initial_topography_model/interface.h>
-#include <aspect/geometry_model/interface.h>
-#include <aspect/gravity_model/interface.h>
-#include <aspect/boundary_temperature/interface.h>
-#include <aspect/boundary_composition/interface.h>
-#include <aspect/initial_temperature/interface.h>
-#include <aspect/initial_composition/interface.h>
-#include <aspect/boundary_velocity/interface.h>
-#include <aspect/boundary_traction/interface.h>
-#include <aspect/mesh_refinement/interface.h>
-#include <aspect/postprocess/interface.h>
-#include <aspect/heating_model/interface.h>
-#include <aspect/adiabatic_conditions/interface.h>
```

Comment:
Should all of these move to `simulator_access.cc`? Or do we skip the inclusion in this case, because `simulator_access.cc` always will include `simulator.h`, which in turn includes all of these?